### DISCUSSION TITLE
Add steam Wishlist

### DISCUSSION THREAD

**Discussion by brenoprata10** [dersvenhesse/awesome-scriptable#6] [Ideas]
Please add my widget to the list, maybe you should create a list for `Gaming` :)
https://twitter.com/BrenoXDDD/status/1358729670221561856?s=20

**Comment by dersvenhesse** [comment]
I like your widget, so I added it.

**Comment by crazygamerdude17** [comment]
> Please add my widget to the list, maybe you should create a list for `Gaming` :)
> https://twitter.com/BrenoXDDD/status/1358729670221561856?s=20

Hey, I love your widget, but, I have no idea how to set it up, when I put an id in, it gave me an when I used it

**Reply by crazygamerdude17** [reply]
Could you help?

**Reply by brenoprata10** [reply]
Hello man, please make sure your wishlist is public on steam and make sure you're adding the correct steam id, it should be something like this: 76561198008818777

**Reply by crazygamerdude17** [reply]
Do I change the first set of numbers or do I change the part where it says {userid} or both?

**Reply by crazygamerdude17** [reply]
I found out how, I just had private wishlist, you made an awesome widget, great job
